### PR TITLE
Feat: Auth

### PR DESCRIPTION
**Pull request by arulk8**
login: https://wonderful-rosalind-def6d1.netlify.app/auth/
sign up: https://wonderful-rosalind-def6d1.netlify.app/auth/signup.html

### CODE REVIEW

File: auth/index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/739eee4960.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="../layout.css" />
+    <link rel="stylesheet" href="../style.css" />
+    <link rel="stylesheet" href="./signin.css" />
```
Comment (krgarima) [inline auth/index.html:13]: Import all CSS files in a single CSS file and give a link of that single file only.

File: auth/signup.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/739eee4960.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="../layout.css" />
+    <link rel="stylesheet" href="../style.css" />
+    <link rel="stylesheet" href="./signin.css" />
```
Comment (krgarima) [inline auth/signup.html:13]: same

**Review by krgarima — COMMENTED** [review]
Styling suggestion: You can give some top and bottom margin to button like 10px. Rest looks good to me.

**Review by Balakumar3018 — APPROVED** [review]
Everything looks good to me

**Review by jatinmore — APPROVED** [review]
Looks good to merge

File: auth/signup.html (hunk 2 above)
Comment (arulk8) [reply auth/signup.html:13]: Will do the change.

**Pull request merged**